cypress/support/page_objects/sliderMove.js

```diff
@@ -0,0 +1,36 @@
+export class SliderMove {
+
+    movingSlider() {
+
+        const currentValueMin = 0;
+        const targetValueMin = 3;
+        const currentValueMax = 5;
+        const targetValueMax = 3;
```

Comment:
I think some of these values could be used as method params, so the moving method could be used in multiple tests with different values. But it would require to modify the rest of the code, so maybe it's not for now.